### ISSUE TITLE
[v4] Cannot adjust bottom sheet position when VoiceOver is enabled 

### ISSUE THREAD

**Issue by virginiab10** [gorhom/react-native-scroll-bottom-sheet#1932]
# Bug

On our react native app we have a bottom sheet where we display an audio player. We are setting one snap points at 75% and when you move the handle to the bottom a new "compact" player appears. The issue is that, despite there is an accessibility message that says what to do to adjust the bottomSheet, that's not working so I can't dismiss the player.
On the other hand, on Android users can slide the bottom sheet to the desired snap point with the two fingers when using TalkBack, the case is not the same with VoiceOver on iOS. While the bottom sheet has an 'adjustable' role the gestures do not seem to do anything at all.

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 4.6.4   |
| react-native                    | 0.73.2   |
| react-native-reanimated         | 3.6.0   |
| react-native-gesture-handler    | 2.14.0  |

## Steps To Reproduce

On iOS:

1. Have a Bottom Sheet and add some snap points (or just one)
2. Enable VoiceOver and set the focus on the Bottom Sheet
3. There is no way to adjust the Bottom Sheet position (you can't dismiss the bottom sheet)

Describe what you expected to happen:

Should be able to adjust the bottom sheet position

## Reproducible sample code

```
import BottomSheet from '@gorhom/bottom-sheet';
import React from 'react';

function App(): React.JSX.Element {
  const snapPoints = useMemo(() => ['75%'], [])
  return (
   <View>
    <BottomSheetModalProvider>
        <BottomSheet
        ref={bottomSheetModalRef}
        index={-1}
        snapPoints={snapPoints}
        enablePanDownToClose={true}
        >
           <BottomSheetScrollView
              style={{backgroundColor: 'white'}}
              contentContainerStyle={styles.extendedPlayerContainer}
            >
               <Text>Bottom sheet test</Text>
           </BottomSheetScrollView>
        </BottomSheet>
      </BottomSheetModalProvider>
      </View>
  );
}

export default App;
```

**Comment by snoronen** [comment]
can we open the issue since it's still there?

**Comment by ahmedu007** [comment]
Hey @virginiab10 
Did you ever get around to finding a solution for this?
